Assets/_BuildingBlocks/Credentials/Scripts/StoredAuthorization/Key.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Netherlands3D.Credentials.StoredAuthorization
+{
+    [Serializable]
+    public class Key : InferableSingleKey
+    {
+        public override AuthorizationType AuthorizationType => AuthorizationType.Key;
+        public override string queryKeyName { get; protected set; } = "key";
```

Comment:
Is protected set nodig? Als het mutabel is kan het misbruikt worden

Reply:
dat is omdat de inheritors van deze class (of later de `QueryStringBasedAuthorization`) hun eigen keyName moeten setten ("token", "key", of "code")

Reply:
maar als het QueryStringBasedAuthorization wordt, kan deze abstract worden en is de protected niet meer nodig

Reply:
Comment staat nog, zeker nu er geen inheritors van deze class zouden moeten zijn (we kunnen zelfs debatteren of deze niet sealed moet zijn; maar daar maken we momenteel nog niet veel gebruik van)